test/e2e/e2e_test.go

```diff
@@ -50,15 +62,26 @@ var _ = Describe("Manager", Ordered, func() {
 	// and deploying the controller.
 	BeforeAll(func() {
 		By("creating manager namespace")
-		cmd := exec.Command("kubectl", "create", "ns", namespace)
+		cmd := exec.Command("kubectl", "create", "ns", operatorNamespace)
 		_, err := utils.Run(cmd)
-		Expect(err).NotTo(HaveOccurred(), "Failed to create namespace")
+		Expect(err).NotTo(HaveOccurred(), "Failed to create operator namespace")
 
-		By("labeling the namespace to enforce the restricted security policy")
-		cmd = exec.Command("kubectl", "label", "--overwrite", "ns", namespace,
+		By("labeling the operator namespace to enforce the restricted security policy")
+		cmd = exec.Command("kubectl", "label", "--overwrite", "ns", operatorNamespace,
 			"pod-security.kubernetes.io/enforce=restricted")
 		_, err = utils.Run(cmd)
-		Expect(err).NotTo(HaveOccurred(), "Failed to label namespace with restricted policy")
+		Expect(err).NotTo(HaveOccurred(), "Failed to label operator namespace with restricted policy")
+
+		By("labeling the operator namespace to be ignored by the webhook")
+		cmd = exec.Command("kubectl", "label", "--overwrite", "ns", operatorNamespace,
```

Comment:
Is this from the operator-sdk scaffolding?
Maybe we should migrate in the future to something more common then `os.exec` for e2e tests?